packages/date-helpers/src/index.ts

```diff
@@ -145,13 +144,13 @@ export const convertDateTimeZone = (
 
 // Convert a DateTime scalar to a Date by stripping off the time
 export const convertDateTimeToDate = (
-  d: Scalars["DateTime"],
-): Scalars["Date"] => {
+  d: Scalars["DateTime"]["input"],
```

Comment:
Huh, this is interesting.  The new codegen has separate input and output scalar types?

Reply:
Yeah! And we can define them depending on what we need to pass in or get out of the API 😁 . So if we wanted to, i think we could do like:

```ts
{
  DateTime: {
    input: string;
    output: Date;
  }
}
```